src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -96,17 +99,31 @@ public ParameterDefinition getParameterDefinition(String name) {
     }
 
     public Set<String> getUsersAsSet() {
+        return getUsersAsSet(null);
+    }
+
+    public Set<String> getUsersAsSet(AbstractBuild<?,?> build) {
         if (users == null || users.equals(""))
             return Collections.emptySet();
 
         Set<String> set = new HashSet<String>();
         for (String user : users.split(",")) {
             user = user.trim();
 
-            if (user.trim().length() > 0)
+            LOGGER.log(Level.FINER, "ManualCondition.getUsersAsSet(): user={0}", user);
+            if (user.startsWith("$")){
+                user = user.substring(1);
+                if (user.startsWith("{") && user.endsWith("}"))
+                    user = user.substring(1, user.length() -1);
+                LOGGER.log(Level.INFO, "ManualCondition.getUsersAsSet(): build.getBuildVariableResolver().resolve(user=${0})={1}", 
```

Comment:
`Level.INFO` is visible by default in Jenkins, so I would move this and the other call to `LOGGER.log` to `Level.FINER` or similar.